documentation/configuration/index.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: documentation
+menu: configuration
+menu: configuration
+pill: configuration
+subtitle: configuration
+---
+
+# Configuration
+
+Flyway has many different parameters that can be set to configure its behavior. These parameters can be set through a variety of different means, depending on how you are using Flyway.
+
+## Usage
+
+### Command Line
+If using the command line, config parameters can be set via command line arguments (e.g. `./flyway -url=jdbc:h2:mem;flyway info`), [configuration files](/documentation/configfiles), or environment variables (e.g. `FLYWAY_URL=jdbc:h2:mem;flyway`).
+
+### Api
+If using the api, config parameters can be set via calling methods on the configuration object returned by `Flyway.configure()` (e.g. `Flyway.configure().url("jdbc:h2:mem;flyway").load()`), [configuration files](/documentation/configfiles), or environment variables if the `.envVars()` method is called on the configuration object.
+
+### Maven
+If using maven, config parameters can be set on the configuration xml block in the maven config, [configuration files](/documentation/configfiles), or environment variables (e.g. `FLYWAY_URL=jdbc:h2:mem;flyway`).
+
+### Gradle
+If using maven, config parameters can be set in the plugin configuration block, [configuration files](/documentation/configfiles), or environment variables (e.g. `FLYWAY_URL=jdbc:h2:mem;flyway`).
+
+## Parameters
+
+### Basic
+- [url](/documentation/configuration/url)
+- [user](/documentation/configuration/user)
+- [password](/documentation/configuration/password)
+- [locations](/documentation/configuration/locations)
+- [driver](/documentation/configuration/driver)
+- [callbacks](/documentation/configuration/callbacks)
+- [configFiles](/documentation/configuration/configFiles)
+- [configFileEncoding](/documentation/configuration/configFileEncoding)
+
+### Advanced
+- [connectRetries](/documentation/configuration/connectRetries)
+- [encoding](/documentation/configuration/encoding)
+- [group](/documentation/configuration/group)
+- [initSql](/documentation/configuration/initSql)
+- [installedBy](/documentation/configuration/installedBy)
+- [jarDirs](/documentation/configuration/jarDirs)
+- [mixed](/documentation/configuration/mixed)
+- [outOfOrder](/documentation/configuration/outOfOrder)
+- [resolvers](/documentation/configuration/resolvers)
+- [skipDefaultCallbacks](/documentation/configuration/skipDefaultCallbacks)
+- [skipDefaultResolvers](/documentation/configuration/skipDefaultResolvers)
+- [table](/documentation/configuration/table)
+- [tablespace](/documentation/configuration/tablespace)
+- [target](/documentation/configuration/target)
+- [validateMigrationNaming](/documentation/configuration/validateMigrationNaming)
+- [validateOnMigrate](/documentation/configuration/validateOnMigrate)
+- [workingDirectory](/documentation/configuration/workingDirectory)
+
+### Flyway Teams
+{% include teams.html %}
```

Comment:
Having both the teams header and the include looks a bit weird, maybe we only need one or the other

![image](https://user-images.githubusercontent.com/17652619/94433875-d9725800-0190-11eb-9850-86f27d6c2087.png)